README.md

```diff
@@ -1 +1,102 @@
-# Apache Ignite 3.x
+Apache Ignite 3.x
+===
+
+Apache Ignite 3 is a distributed database for high-performance computing with in-memory speed.
+
+* MULTI-TIER STORAGE: Ignite scales up and out across memory and disk. By default, Ignite operates in a pure in-memory mode. But, by toggling a single configuration setting, you can turn a cluster into a database that can grow beyond the cluster's memory capacity.
```

Comment:
Ignite 3 is not a multi-tier storage. Actually Ignite 2 too :) The tired storage was a part of Ignite 1 architecture.

And I'm not sure that the in-memory mode is a default mode.